oxenmq/connections.h

```diff
@@ -68,6 +68,9 @@ struct ConnectionID {
     // Returns a copy of the ConnectionID with the route set to empty.
     ConnectionID unrouted() { return ConnectionID{id, pk, ""}; }
 
+    const std::string& to_string() const;
```

Comment:
```suggestion
    std::string to_string() const;
```
No reference to return here.